lib/json_spec/cucumber.rb

```diff
@@ -18,6 +18,18 @@
   end
 end
 
+Then /^the (?:JSON|json)(?: response)?(?: at "(.*)")? with UUIDs should( not)? be:$/ do |path, negative, json|
+  uuid_regexp = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/
+  time_regexp = /[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\.[0-9]{3}Z/
+  last_json_stripped_uuids = last_json.gsub(uuid_regexp, 'UUID').gsub(time_regexp, 'TIME')
+  expected_json_stripped_uuids = json.gsub(uuid_regexp, 'UUID').gsub(time_regexp, 'TIME')
```

Comment:
I understand your surprise... this is my usage.

https://github.com/venuenext/sports-data/blob/19e84e90a074fbde8b74f955e517bc3d9352a055/features/scenarios/api/games.feature#L40-L62

with 3 uuids and 2 time to stub, I found this would complicate the test unnecessarily.
so this is the solution I found to be the easiest and I am still testing what I wanted with my scenario.